READMEs/devops/deploy.md

```diff
@@ -0,0 +1,26 @@
+# Deploy Process
```

Comment:
How do you feel about changing this readme to `EKS.md` and this line to `EKS Deploy Process`? I just feel like `deploy` is a loaded term that could apply to a number of different things (EKS, deploying released content, etc)